File: PSP/Proyectos/LoginSpring_AdrianSaavedra/src/main/java/org/example/loginspring_adriansaavedra/ui/utils/AuthenticationResponse.java
package org.example.loginspring_adriansaavedra.ui.utils;

import lombok.AllArgsConstructor;
import lombok.Builder;
import lombok.Data;
import lombok.NoArgsConstructor;

@Data
@Builder
@AllArgsConstructor
@NoArgsConstructor
public class AuthenticationResponse {
    private Integer userId;
    private String accessToken;
    private String refreshToken;
}

File: MOBILES/FinalExamenMoviles_AdrianSaavedra/app/src/main/java/com/example/finalexamenmoviles_adriansaavedra/data/remote/utils/AuthenticationResponse.kt
package com.example.finalexamenmoviles_adriansaavedra.data.remote.utils


import com.google.gson.annotations.SerializedName

data class AuthenticationResponse(
    @SerializedName("access_token")
    val accessToken: String,
    @SerializedName("refresh_token")
    val refreshToken: String,
)